packages/datadog-plugin-child_process/src/scrub-cmd-params.js

```diff
@@ -63,7 +63,7 @@ function scrubChildProcessCmd (expression) {
 
       if (token === null) {
         continue
-      } else if (typeof token === 'object') {
+      } else if (typeof token === 'object') { // eslint-disable-line @datadog/safe-typeof-object
```

Comment:
Seems like a limitation of the plugin to me, as there is a check for `null`. If that check is removed for any reason, because of this comment the rule won't catch the now invalid code.